### PR TITLE
clarify ratelimit fitler docs

### LINKED ISSUE #1032: clarify service ratelimit
Service `ratelimit()` is not a per backend ratelimit, which is not yet implemented.
We should clarify this in the docs.

### PR DESCRIPTION
**Pull request by szuecs**
fix #1032
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: docs/reference/filters.md [hunk 1]
```diff
@@ -880,7 +880,7 @@ See also the [ratelimit docs](https://godoc.org/github.com/zalando/skipper/ratel
 ## ratelimit
 
 Per skipper instance calculated ratelimit, that allows number of
-requests to a backend. You need to run skipper
+requests to the backend group. You need to run skipper
```
Comment (aryszka) [inline docs/reference/filters.md]: now: "...that allows number of requests to the backend group."
could be: "...that allows forwarding a number of requests to the backend group."

Comment (szuecs) [reply docs/reference/filters.md]: thanks, change pushed

**Comment by aryszka** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**